lib/topological_inventory/ansible_tower/operations/core/sources_api_client.rb

```diff
@@ -0,0 +1,85 @@
+require "sources-api-client"
+
+module TopologicalInventory
+  module AnsibleTower
+    module Operations
+      module Core
+        class SourcesApiClient < ::SourcesApiClient::ApiClient
+          INTERNAL_API_PATH = '//internal/v1.0'.freeze
```

Comment:
Why two leading `/` ?

Reply:
I used the same style as sources-client does: https://github.com/ManageIQ/sources-api-client-ruby/blob/master/lib/sources-api-client/configuration.rb#L133
...but it's probably not necessary